x-pack/plugins/ml/common/types/data_frame_analytics.ts

```diff
@@ -85,3 +85,37 @@ export interface DataFrameAnalyticsConfig {
 }
 
 export type DataFrameAnalysisConfigType = typeof ANALYSIS_CONFIG_TYPE[keyof typeof ANALYSIS_CONFIG_TYPE];
+
+interface ProgressSection {
+  phase: string;
+  progress_percent: number;
+}
+
+enum DATA_FRAME_TASK_STATE {
```

Comment:
Looks like this is a duplicate of `DATA_FRAME_TASK_STATE` in `x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/data_frame_task_state.ts`.  Could this be an opportunity to get rid of the `enum` and refactor it to be the same style as `ANALYSIS_CONFIG_TYPE/DataFrameAnalysisConfigType `?

Reply:
Good call - updated in 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9